tests/test_json_encoding.py

```diff
@@ -3,11 +3,16 @@
 from dataclasses import asdict, dataclass
 from functools import partial
 from json import dumps as sdumps
+from typing import Dict
 
 import pytest
 
+from packaging import version
```

Comment:
No need to add a dependency. Just parse the version by hand. We do it in a couple of places.

Reply:
`packaging` is used by setuptools so I thought it is everywhere. And sure, I can manually parse it.